### ISSUE TITLE
Initiate 508 audit for 526EZ confirmation email - Accessibility

### ISSUE THREAD

**Issue by toddstanich** [department-of-veterans-affairs/notification-api#87]
This is a process card to initiate the 508 audit review within the VA for the 526EZ confirmation email

Resources:
- [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md)
- [VA 508 Site](https://www.section508.va.gov/)

**Comment by lingtran** [comment]
For reference, previous work on accessibility: https://app.zenhub.com/workspaces/vanotify-5ea0ae59b51ccc3c83a8b65b/issues/department-of-veterans-affairs/notification-api/141

**Comment by jsmithVA** [comment]
Here are some helpful links I have found:

[Best Practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md)

[Form to request audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=1Copenut&labels=508%2FAccessibility%2C+launch+review%2C+product+support&template=508-pre-launch-review-template.md&title=Request+accessibility%2F508+review+for+ENTER_PRODUCT_NAME)

**Comment by miabecker** [comment]
Linsey DeLisle - we have a template that we are using to send messages. linsey.delisle@va.gov

**Comment by lingtran** [comment]
Accessibility slack thread convo we started: https://dsva.slack.com/archives/CBU0KDSB1/p1608323618144800
